README.md

```diff
@@ -211,6 +211,19 @@ class AddDemoCulumnToOxUserMigration extends oxMigrationQuery
 *Note: It is better to use generator for migration queries creation*
 
 
+## Run Module Migration
+*Note: Currently this is experimental feature*
```

Comment:
I think this is even better: `*Note: Currently this is an experimental feature*`, sorry for not spotting this in the first review.

Reply:
Done. 
Sorry it was my mistake initially.